x/wasm/types/authz.go

```diff
@@ -15,7 +16,10 @@ import (
 	authztypes "github.com/cosmos/cosmos-sdk/x/authz"
 )
 
-const gasDeserializationCostPerByte = uint64(1)
+const (
+	gasDeserializationCostPerByte = uint64(1)
+	codehashWildcard              = "*"
```

Comment:
let's make the wildcard public so that it can be used from other packages if needed